email.rst

```diff
@@ -41,7 +41,7 @@ environment variable in the ``.env`` file:
 
     # .env (or override MAILER_URL in .env.local to avoid committing your changes)
 
-    # use this to disable email delivery
+    # use this to disable email delivery (this configuration can be used in a test environment)
```

Comment:
Thank you for this proposal, but not only a test but also e dev environment.

I am against merging this comment, sorry.

Reply:
that's why i wrote : "can be". To explain that we can use this configuration on test, for example.